### PR TITLE
update readthedocs

### PR DESCRIPTION
**Pull request by oskooi**
Minor changes to the readthedocs including a reordering of page headings on the left sidebar (to be consistent with the previous Wiki on ab-initio), updates to the installation page for Ubuntu and macOS, etc.

### CODE REVIEW

**Comment by oskooi** [comment]
Changes have been made based on last week's discussion in addition to more tweaks and fixes.

File: mkdocs.yml [hunk 1]
```diff
@@ -24,35 +24,37 @@ markdown_extensions:
...
- - Scheme_Tutorials/Casimir_Forces.md
- - Scheme_Tutorials/Local_Density_of_States.md
- - Scheme_Tutorials/Optical_Forces.md
- - Scheme_Tutorials/Near_to_Far_Field_Spectra.md
- - Scheme_Tutorials/Multilevel_Atomic_Susceptibility.md
+- Download:        'Download.md'
+- Release Notes:   'Release_Notes.md'
+- FAQ:             'FAQ.md'
+- Meep manual:     'index.md'
+- Introduction:
```
Comment (stevengj) [inline mkdocs.yml]: Maybe better to organize into "features", "installation", "scheme interface", "c++ interface" sections

And put all the scheme stuff together

**Comment by oskooi** [comment]
The sidebar groupings have been slightly reorganized along with additional tweaks and fixes.

File: mkdocs.yml [hunk 2]
```diff
@@ -24,37 +24,38 @@ markdown_extensions:
...
+    - Materials:     'Materials.md'
+    - Exploiting Symmetry: 'Exploiting_Symmetry.md'
+    - Cylindrical Coordinates: 'Cylindrical_Coordinates.md'
+    - Perfectly Matched Layers: 'Perfectly_Matched_Layer.md'
+    - Parallel Meep:        'Parallel_Meep.md'
+    - Synchronizing the Magnetic and Electric Fields: 'Synchronizing_the_Magnetic_and_Electric_Fields.md'
+    - Field Function Examples: 'Field_Function_Examples.md'
+    - Yee Lattice: 'Yee_Lattice.md'
+    - Guile and Scheme Information: 'Guile_and_Scheme_Information.md'
+    - The Run Function is Not a Loop: 'The_Run_Function_Is_Not_A_Loop.md'
```
Comment (stevengj) [inline mkdocs.yml]: These two items should go at the end of the "Scheme Interface" section.

**Comment by oskooi** [comment]
Fixed.

**Pull request merged**